lib/features/home/view/widgets/property_item_view.dart

```diff
@@ -0,0 +1,376 @@
+import 'package:hatspace/strings/l10n.dart';
+import 'package:hatspace/theme/hs_theme.dart';
+
+class PropertyItemView extends StatelessWidget {
+  final PropertyItemData property;
+  const PropertyItemView({required this.property, super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return Column(
+      children: [
+        _PropertyImgsCarousel(
+          photos: property.photos,
+          ownerName: property.ownerName,
+          ownerAvatar: property.ownerAvatar,
+          availableDate: property.availableDate,
+          isFavorited: property.isFavorited,
+        ),
+        Container(
+          padding: const EdgeInsets.symmetric(vertical: HsDimens.spacing8),
+          child: Column(
+            crossAxisAlignment: CrossAxisAlignment.start,
+            children: [
+              Row(
+                children: [
+                  Expanded(
+                      child: Row(
+                    children: [
+                      Flexible(
+                          child: Text(
+                        property.price,
+                        maxLines: 4,
+                        overflow: TextOverflow.ellipsis,
+                        style: Theme.of(context)
+                            .textTheme
+                            .displayMedium
+                            ?.copyWith(fontWeight: FontWeight.w700),
+                      )),
+                      const SizedBox(width: HsDimens.spacing4),
+                      Padding(
+                        padding: const EdgeInsets.only(top: 8),
+                        child: Text(
+                          HatSpaceStrings.current.pw,
+                          maxLines: 2,
+                          overflow: TextOverflow.ellipsis,
+                          style: Theme.of(context)
+                              .textTheme
+                              .bodyMedium
+                              ?.copyWith(color: HSColor.neutral6),
+                        ),
+                      )
+                    ],
+                  )),
+                  const SizedBox(width: HsDimens.spacing12),
+                  Expanded(
+                    child: Padding(
+                      padding: const EdgeInsets.only(top: 8),
+                      child: Row(
+                        mainAxisAlignment: MainAxisAlignment.end,
+                        children: [
+                          SvgPicture.asset(
+                            Assets.icons.eye,
+                            colorFilter: const ColorFilter.mode(
+                                HSColor.neutral6, BlendMode.srcIn),
+                          ),
+                          const SizedBox(
+                            width: HsDimens.spacing4,
+                          ),
+                          Flexible(
+                            child: Text(
+                              '${property.todayViews} ${HatSpaceStrings.current.viewsToday}',
+                              maxLines: 2,
```

Comment:
@hophongit98 What do you mean by this sentence `You add maxLines: 2 because you make the width of "viewcount" equal to "property.price".` ? Basically maxLines doesn't have any thing to do with the equal problem you said above. maxLines is for limiting the amount of lines that Text widget can get.

Reply:
1 line